src/components/index.js

```diff
@@ -8,11 +8,11 @@ export const Tabs = () => (
         Documentation
```

Comment:
Can we remove the "link" bit from here? If it's too much work, nbd, but I notice the cursor is a pointer cursor when we hover over this which isn't ideal when we don't have another tab or link anymore.

Reply:
🎉